### PR TITLE
Change the default log level to `info`, avoiding no output when starting a node

### PR DESCRIPTION
**Pull request by marc-casperlabs**
This removes a common pitfall for new users of the node that are confronted without any output at all when starting it up.

### CODE REVIEW

**Review by Fraser999 — APPROVED** [review]

File: node/src/logging.rs [hunk 1]
```diff
@@ -269,18 +271,24 @@ pub fn init_with_config(config: &LoggingConfig) -> anyhow::Result<()> {
         _ => write!(writer, "; {}={:?}", field, value),
     });
 
+    let filter = EnvFilter::new(
+        env::var(LOG_CONFIGURATION_ENVVAR)
+            .as_deref()
+            .unwrap_or("info"),
```
Comment (goral09) [inline node/src/logging.rs:277]: what about all the logs coming from our dependencies? cc @Fraser999

Comment (marc-casperlabs) [reply node/src/logging.rs:277]: This is 100% the same as setting the `RUST_LOG` variable to `info`, which is our common practice. So while I am not sure what the behavior was before, I assume it is unchanged :)

Comment (Fraser999) [reply node/src/logging.rs:277]: @goral09 I haven't seen any third party crates logging at `info` that I recall.  A couple have `debug` level, but at `info` we should be fine afaik.

Comment (goral09) [reply node/src/logging.rs:277]: Thanks.

**Pull request merged**